src/test/java/seedu/address/logic/parser/IVolunteerParserTest.java

```diff
@@ -50,9 +53,14 @@ public void parseCommand_clear() throws Exception {
 
     @Test
     public void parseCommand_delete() throws Exception {
-        VolunteerDeleteCommand command = (VolunteerDeleteCommand) parser.parseCommand(
+
+        Command command = parser.parseCommand(
                 VolunteerDeleteCommand.COMMAND_WORD + " " + INDEX_FIRST_VOLUNTEER.getOneBased());
-        assertEquals(new VolunteerDeleteCommand(INDEX_FIRST_VOLUNTEER), command);
+        if (command instanceof EventDeleteCommand) {
+            assertEquals(new EventDeleteCommand(INDEX_FIRST_EVENT), command);
+        } else {
+            assertEquals(new VolunteerDeleteCommand(INDEX_FIRST_VOLUNTEER), command);
+        }
     }
 
```

Comment:
This looks really dumb, but it failed without having this if else, please let me know if i did something wrong in other part.